cloud-stream-custom-queue-names/src/main/resources/application.yaml

```diff
@@ -0,0 +1,60 @@
+spring:
+  cloud:
+    function:
+      definition: uppercase;lowercase
+    stream:
+      bindings:
+        uppercase-in-0:
+          destination: manualackqueue
+          binder: solace-broker
+          group: myconsumergroup
+          consumer:
+            concurrency: 1
+        uppercase-out-0:
+          destination: uppercase/topic
+          binder: solace-broker
+        lowercase-in-0:
+          destination: lowercase/topic
+          binder: solace-broker
+          group: myconsumergroup
+          consumer:
+            concurrency: 1
+      binders:
+        solace-broker:
+          type: solace
+          environment:
+            solace:
+              java:
+                host: 'tcp://localhost:55555'
+                msgVpn: default
+                clientUsername: default
+                clientPassword: default
+                connectRetries: -1
+                reconnectRetries: -1
+      solace:
+        bindings:
+          uppercase-in-0:
+            consumer:
+              provisionDurableQueue: true
+              addDestinationAsSubscriptionToQueue: true
+              useGroupNameInQueueName: true
+              useDestinationEncodingInQueueName: false
```

Comment:
i believe this was deprecated and not necessary with the new queueNameExpression